src/screens/level.py

```diff
@@ -351,6 +362,18 @@ def start_map_transition(self):
         self.map_transition.activate()
         self.start_transition()
 
+    def decay_health(self):
+        if not self.player.hp <= 10:
+            if not self.player.bathstat and not self.player.has_goggles:
+                self.overlay.health_bar.apply_damage(HEALTH_DECAY_VALUE)
+            elif (
+                not self.player.bathstat
+                and self.player.has_goggles
+                or self.player.bathstat
+                and not self.player.has_goggles
+            ):
+                self.overlay.health_bar.apply_damage((HEALTH_DECAY_VALUE / 2))
```

Comment:
Overall, this might be made simpler (I say might, because I feel it can, but I don't see how right away, so I might be wrong here)